src/app/course/course-item/course-item.component.ts

```diff
@@ -1,15 +1,27 @@
-import { Component, OnInit } from '@angular/core';
-
-@Component({
-  selector: 'app-course-item',
-  templateUrl: './course-item.component.html',
-  styleUrls: ['./course-item.component.scss']
-})
-export class CourseItemComponent implements OnInit {
-
-  constructor() { }
-
-  ngOnInit(): void {
-  }
-
-}
+import { Component, Input, OnInit, Output, EventEmitter, SimpleChanges, OnChanges } from '@angular/core';
+import { CourseInstance } from '../course';
+
+@Component({
+  selector: 'app-course-item',
+  templateUrl: './course-item.component.html',
+  styleUrls: ['./course-item.component.scss']
+})
+export class CourseItemComponent implements OnChanges {
+  @Input() public course: CourseInstance;
+  @Output() public deleteCourseEmitter = new EventEmitter();

```

Comment:
By convention an output name represents an event and should not have the `Emitter` part. You can make the name more natural by renaming it to just `deleteCourse`
```typescript
@Output() public deleteCourse = new EventEmitter();
```